src/themes/teams/components/Icon/svg/icons/bold.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { TeamsProcessedSvgIconSpec } from '../types'
+
+export default {
```

Comment:
Isn't this basically a duplicate of the one in ProcessedIcons folder? Why do we need the dupes?

Reply:
I'm following the template for exporting Icons: https://github.com/stardust-ui/react/pull/585

@Bugaa92, what's the reason behind duplicating the files?